app/controllers/application_controller.rb

```diff
@@ -1,7 +1,17 @@
+# module for user access
+module Current
+  thread_mattr_accessor :user
+end
+
 # Application Controller
 class ApplicationController < ActionController::Base
   before_action :authenticate_user!
-  def after_sign_in_path_for(_resource)
```

Comment:
@ckjones818 This doesn't break redirection after sign in right?

Reply:
Nope

Reply:
@ckjones818